homeassistant/components/reolink/host.py

```diff
@@ -69,10 +70,9 @@ async def async_init(self) -> bool:
             return False
 
         if not self._api.is_admin:
-            _LOGGER.warning(
-                "User '%s' has authorisation level '%s'. Only admin users can change camera settings! Not everything will work",
-                self._api.username,
-                self._api.user_level,
+            await self.stop()
+            raise UserNotAdmin(
+                f"User '{self._api.username}' has authorisation level '{self._api.user_level}', only admin users can change camera settings"
```

Comment:
We normally use American English.
```suggestion
                f"User '{self._api.username}' has authorization level '{self._api.user_level}', only admin users can change camera settings"
```

Reply:
Sorry I am not native english, so I am not good in spelling....